_posts/2015-02-13-python-socketserver.md

```diff
@@ -16,6 +16,12 @@ tags: python server
 
 Graduate student of Paul's at UW-Madison.
 
+### Git Aside: First Steps with a Repository
+
+Matthew Gidden will provide a short, live demo of cloning and updating a
+repository. Most of the content will cover similar examples from Software
+Carpentry workshops ([1][git-local], [2][git-github]]).
```

Comment:
ping

On Tue, Feb 10, 2015 at 2:40 PM, Patrick Shriwise <notifications@github.com>
wrote:

> In _posts/2015-02-13-python-socketserver.md
> <https://github.com/Pshriwise/wisconsin/pull/1#discussion_r24448391>:
>
> > @@ -16,6 +16,12 @@ tags: python server
> >
> >  Graduate student of Paul's at UW-Madison.
> >
> > +### Git Aside: First Steps with a Repository
> > +
> > +Matthew Gidden will provide a short, live demo of cloning and updating a
> > +repository. Most of the content will cover similar examples from Software
> > +Carpentry workshops ([1][git-local], [2][git-github]]).
>
> Extra bracket on the rhs here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Pshriwise/wisconsin/pull/1/files#r24448391>.
>



-- 
Matthew Gidden
Ph.D. Candidate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192